### PR TITLE
Us 03

### PR DESCRIPTION
**Pull request by MaaaChu**

### CODE REVIEW

File: SQLstructure.sql [hunk 1]
```diff
@@ -7,77 +7,98 @@ CREATE TABLE role (
...
-("Account Director", "Recognised leader and developer of Kainos talent, continuously improves Kainos, comfortable in novel situations, owns multiple initiatives, accountable for delivery at Programme level.", null),
-("Account Manager", "Serves the company's commercial and delivery interests, owns single initiatives or projects, advocates effective coaching and ensures that it happens.", null),
-("Junior Account Manager", "Expert in their field, is consulted by others, supervises others, works well even if under pressure, effectively coaches people.", null),
+CREATE TABLE capability (
+	capability_id INT unsigned auto_increment,
+    capability_name varchar(60),
+    PRIMARY KEY(`capability_id`)
+);
+
+INSERT INTO capability (capability_name)
```
Comment (matthewdoreilly) [inline SQLstructure.sql]: Remember to use the data (Job Families, Capabilities, Roles) within https://kainossoftwareltd.sharepoint.com/people/Shared%20Documents/Forms/AllItems.aspx?id=%2Fpeople%2FShared%20Documents%2FJob%20Descriptions. I wouldn't use the existing Career Lattice for reference.

File: klattice/src/app/app.component.html [hunk 2]
```diff
@@ -6,39 +6,69 @@ <h1 class="mainTitle">
...
+                class="roleCap"
+                *ngIf="role.capability_id === capability.capability_id"
+                (click)="
+                  openModal(
+                    content,
+                    role.role_id,
+                    role.role_name,
+                    role.role_summary,
+                    role.role_sum_link
+                  )
```
[the remark this answers is not in the document]
Comment (Broom94) [reply klattice/src/app/app.component.html]: done

File: klattice/src/app/app.component.html [hunk 3]
```diff
@@ -6,39 +6,69 @@ <h1 class="mainTitle">
...
   </div>
   <div class="modal-body">
     <form>
       <div class="form-group">
         <h5 class="jobSpec">Job Specification</h5>
-        <p>{{roleSummary}}</p>
-        <br>
-        <a href=roleLink>Sharepoint Link</a>
+        <p>{{ roleSummary }}</p>
+        <br />
```
[the remark this answers is not in the document]
Comment (Broom94) [reply klattice/src/app/app.component.html]: done

File: klattice/src/app/app.component.ts [hunk 4]
```diff
@@ -9,7 +9,7 @@ import {NgbModal} from '@ng-bootstrap/ng-bootstrap';
 })
 
 export class AppComponent {
-
+
```
[the remark this answers is not in the document]
Comment (Broom94) [reply klattice/src/app/app.component.ts]: done

**Pull request merged**